src/python-common/ceph/deployment/hostspec.py

```diff
@@ -1,9 +1,11 @@
 from collections import OrderedDict
 import errno
 import re
+import sys
+import logging
 from typing import Optional, List, Any, Dict
 
-
+sys.tracebacklimit = 0
```

Comment:
should this be done here, or within the function where we want to avoid the traceback? I can't tell if this will affect other commands/functions if set at the top of the file like this

Reply:
I thought of fixing the same issue in assert_valid_oob , but that's not expected from this PR. I have added it inside the function.